easybuild/easyblocks/g/gcc.py

```diff
@@ -1043,4 +1048,42 @@ def make_module_req_guess(self):
             'LD_LIBRARY_PATH': ['lib', 'lib64'],
             'MANPATH': ['man', 'share/man']
         })
+
+        # Install rpath wrappers, if built with RPATH enabled
+        if build_option('rpath'):
+            def create_rpath_wrappers(targetdir):
+                set_up_configuration(silent=True)
+                tc = Toolchain(name="system", version="0.1")
+
+                # Expect rpath_filter_dirs and rpath_include_dirs as environment variables of
+                # ,-separated lists by default. We might want to allow different separators.
+                # E.g. ":" could be used to get a similar syntax to PATH and LD_LIBRARY_PATH
+                separator = ','
+                rpath_filter_dirs = None
+                if "RPATH_FILTER_DIRS" in os.environ:
+                    rpath_filter_dirs = os.environ["RPATH_FILTER_DIRS"].split(separator)
+                rpath_include_dirs = None
+                if "RPATH_INCLUDE_DIRS" in os.environ:
+                    rpath_include_dirs = os.environ["RPATH_INCLUDE_DIRS"].split(separator)
+
+                tc.prepare_rpath_wrappers(
+                        rpath_filter_dirs=rpath_filter_dirs,
+                        rpath_include_dirs=rpath_include_dirs
+                        )
+
+                # Find location of rpath wrappers. Tempfile in prepare_rpath_wrappers seems to be
+                # in a parent directory of our current gettempdir()-location
+                wrapperpath = glob.glob(os.path.join(tempfile.gettempdir(), "..", "tmp*", RPATH_WRAPPERS_SUBDIR))[0]
+                move_file(wrapperpath, targetdir)
+
+            rpath_wrapperdir = os.path.join('bin', RPATH_WRAPPERS_SUBDIR)
+            absolute_wrapperdir = os.path.join(self.installdir, rpath_wrapperdir)
+            if not os.path.exists(absolute_wrapperdir):
+                # make_module_req_guess seems to be called multiple times
```

Comment:
This is a sign that `make_module_req_guess` is not the right place for this, it should probably go into `post_install_step` instead.

Using `make_module_req_guess` makes sense for testing (since you can add the wrappers to an existing installation via `--module-only`), but it doesn't really belong there.

Reply:
Yes, this is an artifact of testing. It moved to the `post_install_step` now